themes/root/templates/RecordDriver/EDS/export-ris.phtml

```diff
@@ -0,0 +1,108 @@
+<?
+$items = $this->driver->getItems();
```

Comment:
That makes sense. Regarding the prioritized list, I'm wondering if it would make sense to structure that list as a class property of the record driver -- something like `['method' => [ 'array', 'of', 'prioritized', 'fields' ]`. Then individual methods could look up their prioritized list with $this->prioritizedFields[__METHOD__], and if we needed to get a complete list of all fields that we use in the code, we could easily array_merge all of the method values into a single list. Obviously, this would only be a good approach if we needed easy access to that master list of all fields... but this would be a way to achieve that without having to maintain field lists in multiple places.

Anyway, thanks again for the work on this, and have a good weekend!